# HG changeset patch
# User Fabian Groffen <grobian@gentoo.org>
# Date 1248269159 -7200
# Node ID d41e043fa7751bc0c82ecb767d756aa228f56db9
# Parent  848f08512bf303292a8068d3fa074fda73edacd6
Use CRYPT_BACKEND_GPGME instead of non-existent HAVE_GPGME

--- a/keymap.c	Wed Jul 22 14:43:48 2009 +0200
+++ b/keymap.c	Wed Jul 22 15:25:59 2009 +0200
@@ -47,7 +47,7 @@
  { "postpone",	MENU_POST },
  { "pgp",	MENU_PGP },
  { "smime",	MENU_SMIME },
-#ifdef HAVE_GPGME
+#ifdef CRYPT_BACKEND_GPGME
  { "key_select_pgp",	MENU_KEY_SELECT_PGP },
  { "key_select_smime",	MENU_KEY_SELECT_SMIME },
 #endif

